std/experimental/checkedint.d

```diff
@@ -2660,6 +2660,7 @@ if (isIntegral!T && T.sizeof >= 4)
 
 version(unittest) private struct CountOverflows
 {
+private:
```

Comment:
The symbol (`CountOverflows`) defined in the unittest block is already `private`. Perhaps this is a false positive on DScanner's side.